packages/mobile/features/features.ts

```diff
@@ -94,7 +94,7 @@ const features = {
             enabled: false,
         },
         sendAndReceive: {
-            enabled: false,
+            enabled: true,
```

Comment:
Until we dont have the feature implemented i wouldnt enable it on the base branch, i wouldnt keep it as a developer flag for now